lib/modules/manager/pep621/artifacts.ts

```diff
@@ -0,0 +1,23 @@
+import is from '@sindresorhus/is';
+import { writeLocalFile } from '../../../util/fs';
+import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
+import { processors } from './processors';
+
+export async function updateArtifacts(
+  updateArtifact: UpdateArtifact
+): Promise<UpdateArtifactsResult[] | null> {
+  const { packageFileName, newPackageFileContent } = updateArtifact;
+
+  await writeLocalFile(packageFileName, newPackageFileContent);
```

Comment:
```suggestion
```
@rarkins doesn't the upper layer already to this? i havn't seen such a line in other managers.

Reply:
Composer and Bundler for example do this.

Reply:
we should remove those all. as said, renovate writes all updated package files before calling updateArtifacts.

so this is slowing down renovate unnecessarily.